runtest.js

```diff
@@ -66,5 +70,30 @@ process.env.__CONFIGURATION = JSON.stringify(config);
 // Setting real ARGV
 process.argv = argv;
 
-// Execute Jest. Specific target optional.
-shell.exec(`npm run test ${testPath} ${runInBand}`);
+const isLocal = config.site === "local";
+const infoURL = (isLocal ? "http://" : "https://") + config.site + "my.good-loop.com/build/gitlog.txt";
```

Comment:
Let's factor the my.good-loop.com bit out to a variable at the top of the file. 
We want to keep runtest.js as close as possible to the wwappbase.js/template version.